<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindowClass</class>
 <widget class="QWidget" name="LoginWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>628</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>628</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;欢迎使用本管理系统,请选择你需要使用的功能&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QTextBrowser" name="IntroductionText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DengXian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>271</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QPushButton" name="BackPB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>返回登录界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertInfoPB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>增添学生信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpdateInfoPB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>修改学生信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectInfoPB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查询学生信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteInfoPB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除学生信息(权限4可用)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GradePointRankPB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查看绩点排名(前五十)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProfessionInfoPB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查询专业信息</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
